### DISCUSSION TITLE
Force snake_case on new database fields

### DISCUSSION THREAD

**Discussion by th-bc** [laravel/framework#32867] [General]
When I create a new database table field I want it to be in snake_case: currency_code. Often I make the mistake of using camelCase instead: currencyCode. And it's usually a pain to correct when you finally discover it. 
Suggestion: Could we set a boolean somewhere to specify that a warning should be raised upon running the migrations in the case that the field name doesn't seem to be snake_case?

best regards
Allan Th. Andersen

**Comment by netpok** [comment]
I think this should  be the job of a style checker rather than the migrator and since Laravel doesn't have one it shouldn't be added for this specific case. Also it would need to run on every migration so it's probably an unnecessary overhead.

You should probably look into how could this be added to PHP CS or PhpStan (this is a static analyzer so probably not meant for this).
